server/config/persist_options.go

```diff
@@ -35,6 +35,7 @@ import (
 type PersistOptions struct {
 	// configuration -> ttl value
 	ttl             map[string]*cache.TTLString
+	ttlCancel       map[string]context.CancelFunc
```

Comment:
Why create `cache.TTLString` for each key, instead of use only 1 `cache.TTLString` for all keys?

Reply:
For each configuration, they could have different  ttl.